packages/grafana-data/src/types/data.ts

```diff
@@ -21,7 +21,7 @@ export enum LoadingState {
 }
 
 // Should be kept in sync with grafana-plugin-sdk-go/data/frame_meta.go
-export const preferredVisualizationTypes = ['graph', 'table', 'logs', 'trace', 'nodeGraph'] as const;
+export const preferredVisualizationTypes = ['graph', 'table', 'logs', 'trace', 'nodeGraph', 'rawPrometheus'] as const;
```

Comment:
This feels a bit too specific to be in grafana/data. Would `raw` work and be a preferredVisualizationType that can be reused on other places?

Also, have we updated the `frame_meta.go` file?

Reply:
But if I don't understand this incorrectly it will work for all other panels that adds the preferred visualization as raw. So maybe we only support prometheus right now but I guess that with a bit more generic name (raw) I think more panels would be able to support this in the future.

Regarding the `frame_meta.go` I'm not sure what the process is but I think open a PR in that repo makes sense. However this sounds less than optimal.

Reply:
I'm not sure how this UI could be used for other datasources, this UI specifically formats prometheus dataFrames into a list of strings of the metric names and attributes. I don't see any harm in renaming things to be more generic, but I want to stress that this UI is specific to prometheus, and there's no plan to introduce this UI into other datasources (although the table UI might not be a good default UI for other datasources as well).

Reply:
Sorry for the confusion from my part I just realize that I did a mental blunder 🙈 This `preferredVisualizationTypes` is used both in the panel edit and in explore. Please ignore this suggestion and proceed according to your initial plan.